### PR TITLE
[release-4.14] OCPBUGS-18166: add etcd diskjitter dashboards

### PR DESCRIPTION
**Pull request by Elbehery**
resolves https://issues.redhat.com/browse/ETCD-435 

cc @tjungblu @hasbro17 @wking @simonpasquier

### CODE REVIEW

**Comment by simonpasquier** [comment]
/lgtm
from a jsonnet standpoint, I haven't how the dashboard looks like in the console.

**Comment by Elbehery** [comment]
> /lgtm from a jsonnet standpoint, I haven't how the dashboard looks like in the console.

i will create a cluster and paste a screenshot here 👍🏽

**Comment by tjungblu** [comment]
lgtm as well, does it work? :)

**Comment by Elbehery** [comment]
> lgtm as well, does it work? :)

i am trying to get a cluster with this PR atop since yesterday, it fails always :/ 

the CI does not show any issue related to this PR 

keep trying today ...

**Comment by Elbehery** [comment]
also fyi, there is some refactoring upstream going on https://github.com/etcd-io/etcd/pull/16246

**Comment by Elbehery** [comment]
this is logs of failing cluster creation attempt https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/release-openshift-origin-installer-launch-aws-modern/1689655046511267840

**Comment by Elbehery** [comment]
label tide/merge-method-squash

File: jsonnet/diskjitter_dashboard.json [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+      "points": false,
+      "renderer": "flot",
+      "seriesOverrides": [],
+      "spaceLength": 10,
+      "stack": false,
+      "steppedLine": false,
+      "targets": [
+        {
+          "expr": "histogram_quantile(0.99, irate(etcd_disk_wal_fsync_duration_seconds_bucket{job=\"etcd\"}[5m]))\n",
+          "interval": "",
```
[the remark this answers is not in the document]
Comment (tjungblu) [reply jsonnet/diskjitter_dashboard.json:63]: start with p50 for the middle, then p80/p20 and p99/p1 that should probably be more than enough for already for a chart. And don't forget the labels!

[the remark this answers is not in the document]
Comment (Elbehery) [reply jsonnet/diskjitter_dashboard.json:63]: 👍🏽

File: jsonnet/diskjitter_dashboard.json [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+      "steppedLine": false,
+      "targets": [
+        {
+          "expr": "histogram_quantile(0.5, sum by(instance, le) (irate(etcd_disk_wal_fsync_duration_seconds_bucket{job=\"etcd\"}[5m])))\n",
+          "interval": "",
+          "legendFormat": "",
+          "refId": "A"
+        },
+        {
+          "expr": "histogram_quantile(0.8, sum by(instance, le) (irate(etcd_disk_wal_fsync_duration_seconds_bucket{job=\"etcd\"}[5m])))\n",
```
Comment (Elbehery) [inline jsonnet/diskjitter_dashboard.json:68]: @tjungblu  ok ?

Comment (tjungblu) [reply jsonnet/diskjitter_dashboard.json:68]: you're missing the corresponding lower (1-x) boundaries, but yeah, I assume the direction is correct

**Comment by Elbehery** [comment]
Screenshot 2023-08-17 at 00 15 51

**Comment by Elbehery** [comment]
Screenshot 2023-08-17 at 00 16 15

**Comment by Elbehery** [comment]
@simonpasquier hello ✋🏽 

The dashboards still missing the label !! 

i added `sum by (instance, le)`, isn't this the way to add labels ? 
Screenshot 2023-08-17 at 14 53 31
Screenshot 2023-08-17 at 14 53 49

**Pull request merged**